WARNING: This script cannot be executed from within the docker container.
It must have access to the project root at ../../
make[1]: Entering directory '/raid/J54Y574/nv_mlperf-5.0.5/closed/NVIDIA'
Updating Loadgen
HEAD is now at a09ef11 Add MLC Automation commands (#2115)
make[1]: Leaving directory '/raid/J54Y574/nv_mlperf-5.0.5/closed/NVIDIA'
cd ../../ \
	&& python3.10 closed/Sustainable_Metal_Cloud/build/inference/tools/submission/submission_checker.py --input build/submission --submitter Sustainable_Metal_Cloud  2>&1 \
		| tee build/submission/closed/Sustainable_Metal_Cloud/results/submission_checker_log.txt
[2025-02-28 16:27:46,326 submission_checker.py:2124 ERROR] closed/Sustainable_Metal_Cloud contains files with size greater than 50 MB: ['closed/Sustainable_Metal_Cloud/results/H200-SXM-141GBx8_TRT/retinanet/Offline/accuracy/mlperf_log_accuracy.json', 'closed/Sustainable_Metal_Cloud/results/H200-SXM-141GBx8_TRT/retinanet/Server/accuracy/mlperf_log_accuracy.json']
[2025-02-28 16:27:46,326 submission_checker.py:3167 INFO] ---
[2025-02-28 16:27:46,326 submission_checker.py:3173 INFO] ---
[2025-02-28 16:27:46,326 submission_checker.py:3176 ERROR] NoResults closed/Sustainable_Metal_Cloud
[2025-02-28 16:27:46,326 submission_checker.py:3262 INFO] ---
[2025-02-28 16:27:46,326 submission_checker.py:3263 INFO] Results=0, NoResults=1, Power Results=0
[2025-02-28 16:27:46,326 submission_checker.py:3270 INFO] ---
[2025-02-28 16:27:46,326 submission_checker.py:3271 INFO] Closed Results=0, Closed Power Results=0

[2025-02-28 16:27:46,326 submission_checker.py:3276 INFO] Open Results=0, Open Power Results=0

[2025-02-28 16:27:46,326 submission_checker.py:3281 INFO] Network Results=0, Network Power Results=0

[2025-02-28 16:27:46,326 submission_checker.py:3286 INFO] ---
[2025-02-28 16:27:46,326 submission_checker.py:3288 INFO] Systems=0, Power Systems=0
[2025-02-28 16:27:46,326 submission_checker.py:3292 INFO] Closed Systems=0, Closed Power Systems=0
[2025-02-28 16:27:46,326 submission_checker.py:3297 INFO] Open Systems=0, Open Power Systems=0
[2025-02-28 16:27:46,326 submission_checker.py:3302 INFO] Network Systems=0, Network Power Systems=0
[2025-02-28 16:27:46,326 submission_checker.py:3307 INFO] ---
[2025-02-28 16:27:46,327 submission_checker.py:3309 ERROR] SUMMARY: submission has errors
